src/OfficeHours.ts

```diff
@@ -188,42 +215,46 @@ export default class OfficeHours {
 
-        // Request last close message from Discord
-        channel.fetchMessage(this.data.lastCloseMessage)
-            .then(closeMessage => {
-                // Find all users that raised their hand
-                const reactions = closeMessage.reactions.get("✋");
-                if (reactions) {
-                    const usersToMention = reactions.users.array().filter(user => !user.bot);
-                    if (usersToMention.length > 0) {
-                        channel.send(usersToMention.join(", ") + "\n");
-                    }
-                }
+        try {
+            // Request last close message from Discord
+            const closeMessage = await channel.fetchMessage(this.data.lastCloseMessage);
 
-                this.sendOnThisDayMessage(channel);
-            })
-            .catch(reason => {
-                console.warn("Failed getting last close message: " + reason);
-                this.sendOnThisDayMessage(channel);
-            });
+            // Find all users that raised their hand
+            const reactions = closeMessage.reactions.get("✋");
+            if (reactions) {
+                const usersToMention = reactions.users.array().filter(user => !user.bot);
+                if (usersToMention.length > 0) {
+                    channel.send(usersToMention.join(", ") + "\n");
+                }
+            }
+            this.sendOnThisDayMessage(channel);
+        } catch (error) {
+            console.error(`Error occurred while fetching last close message: ${error}`);
+        }
     }
 
     private async close(channel: Discord.TextChannel) {
         if (!this.data.isOpen) return;
         this.data.isOpen = false;
 
-        const everyone = channel.guild.roles.find("name", "@everyone");
-        await channel.overwritePermissions(everyone, { SEND_MESSAGES: false });
+        try {
+            const everyone = channel.guild.roles.find("name", "@everyone");
+            await channel.overwritePermissions(everyone, { SEND_MESSAGES: false });
+        } catch (error) {
+            console.error(`Error occurred while overwriting permissions: ${error}`);
+        }
 
-        channel.send(this.sharedSettings.officehours.closeMessage.replace(/{botty}/g, this.bot.user.toString()))
-            .then(message => {
-                if (Array.isArray(message)) {
-                    message = message[0];
-                }
+        try {
+            let message = await channel.send(this.sharedSettings.officehours.closeMessage.replace(/{botty}/g, this.bot.user.toString()));
+            if (Array.isArray(message)) {
+                message = message[0];
+            }
 
-                this.data.lastCloseMessage = message.id;
-                message.react("✋");
-            });
+            this.data.lastCloseMessage = message.id;
+            message.react("✋");
+        } catch (error) {
+            console.error(`Error occurred while sending close message: ${error}`);
```

Comment:
You might want to set `lastCloseMessage` to null here so the next time office hours reopens it won't operate on a stale message.

Reply:
Fixed, github wont show as outdated for some reason..